### PR TITLE
Wrapped floats with int()'s and changed labels to targets

### PR DESCRIPTION
**Pull request by ProgramItUp**
This pull fixes issue #108  [https://github.com/carpedm20/DCGAN-tensorflow/issues/108](https://github.com/carpedm20/DCGAN-tensorflow/issues/108)

Following the advice of 
@pbcquoc, @821760408-sp  and others, 
I wrapped floats with int()'s and changed labels to targets.  This fixes the problem and the training now starts to run.

There there is a new bug, with loading checkpoints.  I'll create a new issue on this.

### CODE REVIEW

**Comment by carpedm20** [comment]
Thanks for the fix but I think they're using `logits` in the HEAD of the master https://github.com/tensorflow/tensorflow/blob/131c3a67a7b8d27fd918e0bc5bddb3cb086de57e/tensorflow/python/ops/nn_impl.py#L100 so I think it's better to make things like https://github.com/carpedm20/DCGAN-tensorflow/blob/master/ops.py#L9
